telebot.go

```diff
@@ -144,6 +144,11 @@ const (
 	//
 	// Handler: func(*Message)
 	OnVoiceChatPartecipantsInvited = "\avoice_chat_partecipants_invited"
+
+	// Will fire on ProximityAlertTriggered
+	//
+	// Handler: func(*Message)
+	OnProximityAlertTriggered = "\aproximity_alert_triggered"
```

Comment:
My proposal is to rename it to `OnProximityAlert` and similarly for other related structs or fields, as word _triggered_ is already implied when you handle events.